src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Serializer.cs

```diff
@@ -2250,6 +2466,172 @@ protected virtual void WriteSubjectLocality(XmlWriter writer, Saml2SubjectLocali
             writer.WriteEndElement();
         }
 
+        private EncryptedData CreateEncryptedData(byte[] assertionData, EncryptingCredentials encryptingCredentials, out SymmetricSecurityKey sessionKey)
```

Comment:
>EncryptedData [](start = 16, length = 13)

suggestion: instead of an out parameter you can also return a tuple (if the platforms this runs on support value tuples that might be even better.

Reply:
I was thinking tuples while refactoring this method, and it seemed to me that using out param is a bit cleaner, considering that it works on each platform and that CreateEncryptedData is a private method.
Is there a reason why tuples (not ValueTuples) are preferable over *out* param, for this specific case?